README.md

```diff
@@ -1,18 +1,54 @@
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/google/osv-scanner/badge)](https://api.securityscorecards.dev/projects/github.com/google/osv-scanner)
 
-# OSV-Scanner (Preview)
+# OSV-Scanner
 
-OSV-Scanner is a vulnerability scanner written in Go which uses the data provided by https://osv.dev
+Use OSV-Scanner to find existing vulnerabilities affecting your project's dependencies.
 
-This tool is currently under development and is subject to change.
+OSV-Scanner provides an officially supported frontend to the [OSV database](https://osv.dev/) that connects a project’s list of dependencies with the vulnerabilities that affect them. Since the OSV.dev database is open source and distributed, it has several benefits in comparison with closed source advisory databases and scanners:
+
+- Each advisory comes from an open and authoritative source (e.g. the RustSec Advisory Database)
+- Anyone can suggest improvements to advisories, resulting in a very high quality database
+- The OSV format unambiguously stores information about affected versions in a machine-readable format that precisely maps onto a developer’s list of packages
+
+The above all results in fewer, more actionable vulnerability notifications, which reduces the time needed to resolve them.
 
 ## Installing
 
+### Prerequisites
+Requires go 1.18+ to be installed
+
+### Installation Process
+Run
 ```bash
 $ go install github.com/google/osv-scanner/cmd/osv-scanner@latest
```

Comment:
Should we also add a section on how we will adhere to SemVer and what guarantees there are? 

Also, we'll probably want to replace this @latest with v1 or something too.

Reply:
changed this to v1, I'll push a tag after this PR merged, which I think is all that's needed to create a version?

Also added a line about semver adherence.